spec/features/applications/new_spec.rb

```diff
@@ -31,15 +21,15 @@
 
         expect(page.has_field?).to eq(true)
 
-        fill_in "application[name]", with: "Kaelin"
-        fill_in "application[street_address]", with: "1289 South St"
-        fill_in "application[city]", with: "Denver"
-        fill_in "application[state]", with: "CO"
-        fill_in "application[zipcode]", with: "80207"
+        fill_in "Name:", with: "Kaelin S"
+        fill_in "Address:", with: "1289 South St"
+        fill_in "City:", with: "Denver"
+        fill_in "State:", with: "CO"
+        fill_in "Zipcode:", with: "80207"
 
-        click_button('Submit')
+        click_button('Submit Application')
 
-        expect(current_path).to eq("/applications/#{@application.id}")
+        expect(current_path).to eq("/applications/#{@application.id + 1}")
```

Comment:
Not sure if there's a more appropriate convention between + 1 or .last, we can come back if anything! Great test!